src/main/java/net/countercraft/movecraft/combat/features/tracking/KillCredCommand.java

```diff
@@ -0,0 +1,40 @@
+package net.countercraft.movecraft.combat.features.tracking;
+
+import net.countercraft.movecraft.Movecraft;
+import net.countercraft.movecraft.combat.MovecraftCombat;
+import net.countercraft.movecraft.combat.localisation.I18nSupport;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandExecutor;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+import org.jetbrains.annotations.NotNull;
+
+public class KillCredCommand implements CommandExecutor {
+    private final DamageTracking damageTracking;
+
+    public KillCredCommand(DamageTracking damageTracking) {
+        this.damageTracking = damageTracking;
+    }
+
+    @Override
+    public boolean onCommand(@NotNull CommandSender sender, @NotNull Command command, @NotNull String label, @NotNull String... args) {
+        if (sender instanceof Player) {
+            Player p = (Player) sender;
+            if (p.hasPermission("movecraft.combat.obfuscateKillCredit")) {
+                if (damageTracking.getKillCred(p)) {
+                    damageTracking.flipKillCred(p);
+                    p.sendMessage(I18nSupport.getInternationalisedString("You are no longer hiding your kills."));
+                    return true;
+                } else {
+                    damageTracking.flipKillCred(p);
+                    p.sendMessage(I18nSupport.getInternationalisedString("You are now hiding your kills."));
+                    return true;
+                }
```

Comment:
This could be refactored to flip kill cred, check the new state of the kill cred, let the player know, then return true.